ui/app/hooks/tests/useTransactionDisplayData.test.js

```diff
@@ -121,7 +126,9 @@ const renderHookWithRouter = (cb, tokenAddress) => {
     tokenAddress ? `${ASSET_ROUTE}/${tokenAddress}` : DEFAULT_ROUTE,
   ]
   // eslint-disable-next-line
-  const wrapper = ({ children }) => <MemoryRouter initialEntries={initialEntries}>{children}</MemoryRouter>
+  const wrapper = ({ children }) => (
```

Comment:
this looks to be prettier's doing -- I'll see whats causing this

Reply:
Seems like this should have been done all along -- not quite sure why it wasn't done by --fix on the other pr.